python/docs/examples.rst

```diff
@@ -1,8 +1,205 @@
 Examples
-========
+~~~~~~~~
+
+Connecting to Feldera Sandbox
+=============================
+
+.. code-block:: python
+
+    from feldera import FelderaClient, PipelineBuilder
+
+    client = FelderaClient('https://try.feldera.com', api_key=api_key)
+
+    pipeline = PipelineBuilder(client, name, sql).create()
+
+Connecting to Feldera on localhost
+==================================
+
+.. code-block:: python
+
+    from feldera import FelderaClient, PipelineBuilder
+
+    client = FelderaClient('https://try.feldera.com', api_key=api_key)
+
+    pipeline = PipelineBuilder(client, name, sql).create()
+
+Creating a Pipeline
+===================
+
+.. code-block:: python
+
+    sql = """
+    CREATE TABLE student (
+        name STRING,
+        id INT
+    );
+
+    CREATE TABLE grades (
+        student_id INT,
+        science INT,
+        maths INT,
+        art INT
+    );
+
+    CREATE VIEW average_scores AS SELECT name, ((science + maths + art) / 3) as average FROM {TBL_NAMES[0]} JOIN {TBL_NAMES[1]} on id = student_id ORDER BY average DESC;
+    """
+
+    pipeline = PipelineBuilder(client, name="notebook", sql=sql).create_or_replace()
```

Comment:
```suggestion
    # This will shutdown and overwrite any existing pipeline with the same name.
    pipeline = PipelineBuilder(client, name="notebook", sql=sql).create_or_replace()
```